src/mantine-rte/src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -131,7 +131,7 @@ export const RichTextEditor = forwardRef<Editor, RichTextEditorProps>(
           upload: (file: File) => onImageUpload(file),
         },
       }),
-      [uuid, mentions, externalModules]
+      [uuid, mentions, externalModules, onImageUpload]
```

Comment:
I also got a warning from ESLint about this when working with this file from another project